crates/fj-viewer/src/graphics/navigation_cube.rs

```diff
@@ -0,0 +1,220 @@
+use fj_math::Transform;
+use wgpu::util::DeviceExt;
+
+use super::model::{self, load_model, DrawModel, Model};
+
+#[derive(Debug)]
+pub struct NavigationCubeRenderer {
+    cube_model: Model,
+    render_pipeline: wgpu::RenderPipeline,
+    mvp_matrix_bind_group: wgpu::BindGroup,
+    mvp_matrix_buffer: wgpu::Buffer,
+}
+
+const SCALE_FACTOR: f64 = 0.13;
+const CUBE_TRANSLATION: [f64; 3] = [0.8, 0.7, 0.0];
```

Comment:
It seems those constants are used only in a single method. Maybe it makes more sense to move them into the local scope of that method. That would make their meaning a bit clearer, I think, as they would be closer to the code that uses them.